cfgov/jinja2/v1/_includes/macros/category-slug.html

```diff
@@ -41,7 +41,7 @@
 
     {% call _category_link(href, classes) %}
         {% set cat = category_label(category) or category %}
-        {{ category_icon.render(cat) | safe }}
```

Comment:
I presume this `| safe` can be removed, because `| safe` is called inside the `category_icon.render` macro at https://github.com/cfpb/consumerfinance.gov/pull/6487/files#diff-fa4fbd84659761aa3aa53c85f486c42c6fb05be2072dec8600a6d13958827804R57